documentation/host/data.rst

```diff
@@ -281,6 +309,8 @@ You can visualise the data model like this:
 This will generate a picture based on the model code.
 You can also generate picture based on the actual database, see inside the Makefile. 
 
+.. note:: If you encounter "error: externally-managed-environment" when running `make test`, add "[global] break-system-packages = true" to pip config file (find config - `pip config list -v`)
```

Comment:
Is this advice not potentially breaking people's system setup?

Is this maybe only happening when working outside of a virtual environment? Or what was your situation?

Reply:
Yeah it may break setup, so I changed the advice. I was getting this error in virtual env though)